activesupport/test/abstract_unit.rb

```diff
@@ -2,6 +2,7 @@
 
 ORIG_ARGV = ARGV.dup
 
+require "bundler/setup"
```

Comment:
idk about this one ... there might be a reason this is not loaded, but it is needed when running pure `ruby path/to/test.rb` and afaik it's a best-practice